modmail/database/models/__init__.py

```diff
@@ -0,0 +1,20 @@
+from .attachments import Attachments
+from .configuration import Configurations
+from .embeds import Embeds
+from .emojis import Emojis
+from .messages import Messages
+from .servers import Servers
+from .stickers import Stickers
+from .tickets import Tickets
+
+
+__all__ = (
```

Comment:
need a channel model

Will hold classes that implement both discord.abc.Messageable and discord.GuildChannel

Reply:
Why do we need a channels DB model?